www/generate_benchmark_list.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: William Haack (will@cockroachlabs.com)
+
+function loadTestListFromJSON(callback) {
+    loadJSON("test_names.json", callback);
+}
+
+function getTests(directory) {
```

Comment:
Personally I'd recommend just pulling in `lodash` so you can just use _.values(directory) here, but that's because I love `lodash`.

Since you have jQuery here you could write something like
`getValues = (obj)=> $.map(obj, (v)=>v)`